### ISSUE TITLE
No signifier you can drag and drop to sidebar folder

### ISSUE THREAD

**Issue by D0J0P** [elementary/files#687]
When you drag and drop something to the Trash, it does it, and the Trash icon will fill up, but there's no way of telling before you've done it, as there's no illuminating/highlight on hover when you're dragging a file to the trash or any folder on the sidebar. It would be better if there was one, to show you that you can drop there, and when it's done you know you dropped it there(except it's easy to tell for Trash if it was empty before).

**Comment by eljefuri** [comment]
Agreed. Especially in the absence of spring-loaded folders, this is an extraordinarily important bit of UI feedback.

**Comment by jeremypw** [comment]
There is some visual indication in that a 'plus' item appears to the top left of the drag icon when the item being hovered will accept the file when dropped.  The emblem is black and white when the file will be moved; green and white when it will be copied.  Other possible drag icon emblems are `link` (linked chain) and `ask` (question mark).  There is an issue where the sidebar does not currently draw a line between item when dropping the file will create a new bookmark.  I think this is a stylesheet matter (Files does not draw the line)

**Comment by eljefuri** [comment]
Ah, I had not even noticed that @jeremypw thanks for pointing it out. Still, I think the more appropriate place for an indicator is actually the folder icon and name itself. I'm having difficulty recalling now how it works in other operating systems. I believe that macOS either highlights the name of the folder in blue, or shows an alternative, "open" version of the folder icon, or both.

**Comment by jeremypw** [comment]
Yes, when dragging in a view, the folder icon does change to an "open" version when hovered, but this does not happen in sidebar.  For consistency, it should.

**Comment by jeremypw** [comment]
I'll bear this in mind for the Gtk4 sidebar rewrite.
